### DISCUSSION TITLE
m.tiktok.com vs t.tiktok.com

### DISCUSSION THREAD

**Discussion by reccardt** [davidteather/TikTok-Api#469] [Q&A] [answered]
What's the difference between m.tiktok.com and t.tiktok.com? I know m.tiktok.com is more likely to provoke a captcha, but I get different results when I call trending on them. t,tiktok.com seems to have a much higher percentage of music.original:True.

**Comment by davidteather** [comment, marked as answer]
t.tiktok.com is their test server endpoint, I haven't investigated discrepancies between the endpoint though.
